lib/pharos/transport/command/ssh.rb

```diff
@@ -12,13 +12,13 @@ def hostname
 
         # @return [Pharos::Transport::CommandResult]
         def run
-          raise Pharos::ExecError, "Connection not established" unless @client.connected?
+          raise Pharos::ExecError.new(@source || cmd, -127, "Connection not established") unless @client.connected?
```

Comment:
This and the identical one below have been broken forever, but the only way to reach it is via console or if writing a new command that does not `GatherFacts` before operations.